backend/LexBoxApi/Controllers/LoginController.cs

```diff
@@ -56,6 +58,54 @@ public class LoginController(
         return Redirect(returnTo);
     }
 
+    [HttpPost("signin-google")]
+    [AllowAnonymous]
+    public async Task<ActionResult> GoogleResponse([FromForm] string credential, string? returnTo)
+    {
+        var claimsIdentity = await googleTokenValidator.ValidateGoogleJwt(credential);
+        var email = claimsIdentity?.FindFirst(claim => claim.Type == ClaimTypes.Email);
+        var avatar = claimsIdentity?.FindFirst(claim => claim.Type == "picture");
+        var name = claimsIdentity?.FindFirst(claim => claim.Type == "name");
+        var locale = claimsIdentity?.FindFirst(claim => claim.Type == "locale");
+        ArgumentNullException.ThrowIfNull(email);
+        // ArgumentNullException.ThrowIfNull(avatar);
+        ArgumentNullException.ThrowIfNull(name);
+        // foreach (var claim in claimsIdentity?.Claims ?? [])
+        // {
+        //     Console.WriteLine(claim);
+        // }
```

Comment:
Done in commit 100f3a17. I left in one line of commented-out code for getting avatars from the JWT, because it's not all that easy to find out what claim name Google uses for avatars in its JWTs. Removed the rest.